docs/source/api/v3/deliveryservice_requests_id_assign.rst

```diff
@@ -29,7 +29,7 @@ Assign a :term:`Delivery Service Request` to a user.
 Request Structure
 -----------------
 :id:       The integral, unique identifier assigned to the :term:`DSR <Delivery Service Request>`
-:assignee: The username of the user to whom the :term:`Delivery Service Request` is assigned.
+:assigneeId: The username of the user to whom the :term:`Delivery Service Request` is assigned.
```

Comment:
This description needs to be changed; the value is the user's *ID*, not the username.